osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySpinner.cs

```diff
@@ -26,12 +28,12 @@ public abstract class LegacySpinner : CompositeDrawable
 
         protected const float SPINNER_Y_CENTRE = SPINNER_TOP_OFFSET + 219f;
 
-        protected const float SPRITE_SCALE = 0.625f;
-
         private const float spm_hide_offset = 50f;
 
         protected DrawableSpinner DrawableSpinner { get; private set; }
 
+        public abstract Drawable ApproachCircle { get; protected set; }
```

Comment:
why abstract

Reply:
Whoops... didn't actually mean that one.